capa/render/default.py

```diff
@@ -148,7 +152,60 @@ def render_attack(doc: rd.ResultDocument, ostream: StringIO):
     if rows:
         ostream.write(
             tabulate.tabulate(
-                rows, headers=[width("ATT&CK Tactic", 20), width("ATT&CK Technique", 80)], tablefmt="mixed_grid"
+                rows,
+                headers=[width("ATT&CK Tactic", 20), width("ATT&CK Technique", 80)],
+                tablefmt="mixed_grid",
+            )
+        )
+        ostream.write("\n")
+
+
+def render_maec(doc: rd.ResultDocument, ostream: StringIO):
+    """
+    example::
+
+        +--------------------------+-----------------------------------------------------------+
+        | MAEC Schema              | MAEC Value                                                |
+        |--------------------------+-----------------------------------------------------------|
+        | analysis-conclusion      | Anti-Virus Detected                                       |
+        |--------------------------+-----------------------------------------------------------|
+        | malware-family           | Ransomware                                                |
+        |--------------------------+-----------------------------------------------------------|
+        | malware-category         | Downloader                                                |
+        |                          | Launcher                                                  |
+        +--------------------------+-----------------------------------------------------------+
+    """
+    maec_schema: dict[str, set[str]] = {
+        "analysis-conclusion": set(),
+        "analysis-conclusion-ov": set(),
+        "malware-family": set(),
+        "malware-category": set(),
+        "malware-category-ov": set(),
+    }
```

Comment:
I think I should be using a `collections.defaultdict(set)` here to stay in line with the other `render_*` functions?

Reply:
yes, please